src/init.cpp

```diff
@@ -1979,9 +1977,12 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
 
     connOptions.nLocalServices      = nLocalServices;
     connOptions.nMaxConnections     = nMaxConnections;
-    connOptions.nSendBufferMaxSize  = 1000*GetArg("-maxsendbuffer", 1*1000);
-    connOptions.nReceiveFloodSize   = 1000*GetArg("-maxreceivebuffer", 5*1000);
 
+    unsigned int fromKBtoBfactor = 1000; // from kilobytes to bytes
+    std::vector<std::pair<int64_t, int64_t>> limits = {{1, std::numeric_limits<unsigned int>::max() / fromKBtoBfactor}};
```

Comment:
Very small thing here, but I would suggest being more explicit on the usage/meaning of these variables, and make them `static const`.

Reply:
Made them `const` (I think `static` is not needed being `AppInit2` called at startup only) and renamed more properly.